platforms/hyperledger-besu/configuration/samples/network-besu-new-memberorg.yaml

```diff
@@ -30,6 +30,16 @@ network:
     loadBalancerSourceRanges: # (Optional) Default value is '0.0.0.0/0', this value can be changed to any other IP adres or list (comma-separated without spaces) of IP adresses, this is valid only if proxy='ambassador'
     retry_count: 20                 # Retry count for the checks on Kubernetes cluster
     external_dns: enabled           # Should be enabled if using external-dns for automatic route configuration
+
+  # Prometheus setup section
+  prometheus:
+    enabled: "true"                 # Set to "true" to enable Prometheus monitoring, or "false" to disable it.
```

Comment:
remove double quotes for true so that it acts as boolean

Reply:
Fixed.